fixture-monkey-javax-validation/src/main/java/com/navercorp/fixturemonkey/javax/validation/introspector/JavaxValidationConstraintGenerator.java

```diff
@@ -212,80 +161,91 @@ public JavaIntegerConstraint generateIntegerConstraint(ArbitraryGeneratorContext
 				decimalMax = decimalMax.subtract(BigInteger.ONE);
 			}
 
-			if (decimalMax.compareTo(BigInteger.ZERO) > 0) {
-				if (positiveMax == null) {
-					positiveMax = decimalMax;
-				} else {
-					positiveMax = positiveMax.max(decimalMax);
-				}
-			} else {
-				if (negativeMax == null) {
-					negativeMax = decimalMax;
-				} else {
-					negativeMax = negativeMax.max(decimalMax);
-				}
+			if (max == null || decimalMax.compareTo(max) < 0) {
+				max = decimalMax;
 			}
 		}
 
-		if (context.findAnnotation(Negative.class).isPresent()) {
-			if (negativeMax == null || negativeMax.compareTo(BigInteger.ZERO) > 0) {
-				negativeMax = BigInteger.valueOf(-1);
+		if (context.findAnnotation(Positive.class).isPresent()) {
+			BigInteger positiveMin = BigInteger.ONE;
+			if (min == null || positiveMin.compareTo(min) > 0) {
+				min = positiveMin;
 			}
 		}
 
-		if (context.findAnnotation(NegativeOrZero.class).isPresent()) {
-			if (negativeMax == null || negativeMax.compareTo(BigInteger.ZERO) > 0) {
-				negativeMax = BigInteger.ZERO;
+		if (context.findAnnotation(PositiveOrZero.class).isPresent()) {
+			BigInteger positiveOrZeroMin = BigInteger.ZERO;
+			if (min == null || positiveOrZeroMin.compareTo(min) > 0) {
+				min = positiveOrZeroMin;
 			}
 		}
 
-		if (context.findAnnotation(Positive.class).isPresent()) {
-			if (positiveMin == null || positiveMin.compareTo(BigInteger.ZERO) < 0) {
-				positiveMin = BigInteger.ONE;
+		if (context.findAnnotation(Negative.class).isPresent()) {
+			BigInteger negativeMax = BigInteger.valueOf(-1);
+			if (max == null || negativeMax.compareTo(max) < 0) {
+				max = negativeMax;
 			}
 		}
 
-		if (context.findAnnotation(PositiveOrZero.class).isPresent()) {
-			if (positiveMin == null || positiveMin.compareTo(BigInteger.ZERO) < 0) {
-				positiveMin = BigInteger.ZERO;
+		if (context.findAnnotation(NegativeOrZero.class).isPresent()) {
+			BigInteger negativeOrZeroMax = BigInteger.ZERO;
+			if (max == null || negativeOrZeroMax.compareTo(max) < 0) {
+				max = negativeOrZeroMax;
+			}
+		}
+
+		Optional<Digits> digitsAnn = context.findAnnotation(Digits.class);
```

Comment:
It is better not to use the abbreviation to avoid confusion.

`digitsAnn` -> `digitsAnnotation`